contracts/axelar-gateway/tests/testdata/call_contract.golden

```diff
@@ -1,3 +1,9 @@
-contract: Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M)
-topics: (Symbol(contract_called), Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAITA4), String(ethereum), String(0x4EFE356BEDeCC817cb89B4E9b796dB8bC188DC59), BytesN<32>(86, 87, 13, 226, 135, 215, 60, 209, 203, 96, 146, 187, 143, 222, 230, 23, 57, 116, 149, 95, 222, 243, 69, 174, 87, 158, 233, 244, 117, 234, 116, 50))
-data: (Bytes(18, 52))
\ No newline at end of file
+Contract(CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAHK3M)
```

Comment:
I'm including the contract id to ensure that the event is emitted from the same contract (it's plausible that two contracts emit the same event, e.g. ownership transfer)